source/state_representation/include/state_representation/space/cartesian/CartesianState.hpp

```diff
@@ -679,6 +680,7 @@ inline std::string cartesian_state_variable_to_string(const CartesianStateVariab
     case CartesianStateVariable::ALL:
       return "all";
   }
+  __builtin_unreachable();
```

Comment:
The compile doesn't realize that the switch covers all cases?

Reply:
I'll check again. Perhaps I misread one of the other warnings. But I think surprisingly no